### PR TITLE
python-axolotl-curve25519: Add package

### PR DESCRIPTION
**Pull request by mimi89999**

### CODE REVIEW

File: mingw-w64-python-python-axolotl-curve25519/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Maintainer: Michel Le Bihan <michel@lebihan.pl>
+
+_realname=python-axolotl-curve25519
```
Comment (Biswa96) [inline mingw-w64-python-python-axolotl-curve25519/PKGBUILD:3]:
```suggestion
_realname=axolotl-curve25519
```

**Comment by Biswa96** [comment]
It seems that the repository is not maintained. The last commit in the upstream repository was [on Apr 14, 2018](https://github.com/tgalal/python-axolotl-curve25519/commit/293f9cd5a42459f7e68748764de8387f37147db2)
. Should it be added as a package here?

**Comment by mimi89999** [comment]
> It seems that the repository is not maintained. The last commit in the upstream repository was [on Apr 14, 2018](https://github.com/tgalal/python-axolotl-curve25519/commit/293f9cd5a42459f7e68748764de8387f37147db2) . Should it be added as a package here?

I saw that. The package is used by `python-axolotl` that is used by Gajim, so I think that it should be packaged.

File: mingw-w64-python-python-axolotl-curve25519/PKGBUILD (hunk 1 above)
Comment (mimi89999) [reply mingw-w64-python-python-axolotl-curve25519/PKGBUILD:3]: Shouldn't `_realname` always be the name on PyPi?

Comment (Biswa96) [reply mingw-w64-python-python-axolotl-curve25519/PKGBUILD:3]: There is no rule like that. The issue is that the package name has two 'python' words.

Comment (mimi89999) [reply mingw-w64-python-python-axolotl-curve25519/PKGBUILD]: OK. Renamed.

**Pull request closed without merging**